src/schedule.py

```diff
@@ -24,17 +24,16 @@ def delete_all_schedules_for_ext_id(client: CogniteClient, function_external_id:
 
 
 def deploy_schedule(client: CogniteClient, function: Function, config: FunctionConfig):
-    delete_all_schedules_for_ext_id(client, function.external_id)
-
     if not config.schedules:
-        logger.info("Skipped step of attaching schedules!")
+        logger.info("No schedules to attach!")
         return
 
+    logger.info(f"Attaching {len(config.schedules)} to {function.external_id}")
```

Comment:
might be good to say attaching **schedules** here
```suggestion
    logger.info(f"Attaching {len(config.schedules)} schedules to {function.external_id}")
```

Reply:
nice ty